### DISCUSSION TITLE
Welcome to Widgetbook Discussions!

### DISCUSSION THREAD

**Discussion by jenshor** [widgetbook/widgetbook#691] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rasha-Mosaed** [comment]
I'm a junior in flutter and widgetbook too. 
but i love it , and start to add widgetbook to our app in my company. 

our app is very very huge and i face many challenges , but am okay with this community :)

my hope is to add valuable to grate widgetbook project

**Reply by LucasJosefiak** [reply]
Thanks for sharing! Amazing that you're using Widgetbook and engaging so much with the community!🥳

**Comment by dmytro-kravchyna** [comment]
- Does Widgetbook support Figma comparison in Azure DevOps?
- Does it include some review flow: notes, markers, etc?

**Reply by LucasJosefiak** [reply]
1. Yes
2. Yes

I'm happy to give you a demo! Please feel free to choose a time slot in [my calendar](https://meetings-eu1.hubspot.com/lucas-josefiak)!🙂

**Comment by AcousticDevops** [comment]
As a beginner to widgetbook, I will appreciate having a tutorial about setting up widgetbook. I was going through the docs, but was running into issues setting it up. At first I did not even know it had to be inside at the root level of the main app. Thanks
